go.mod

```diff
@@ -34,21 +35,23 @@ require (
 	github.com/go-playground/locales v0.14.1 // indirect
 	github.com/go-playground/universal-translator v0.18.1 // indirect
 	github.com/h2non/parth v0.0.0-20190131123155-b4df798d6542 // indirect
-	github.com/klauspost/cpuid/v2 v2.0.9 // indirect
+	github.com/hyperledger/aries-framework-go/spi v0.0.0-20221025204933-b807371b6f1e // indirect
+	github.com/kilic/bls12-381 v0.1.1-0.20210503002446-7b7597926c69 // indirect
+	github.com/kr/text v0.2.0 // indirect
```

Comment:
This looks sus haha. Who's using this?

Reply:
`github.com/hyperledger/aries-framework-go@v0.1.9-0.20230217102417-a948231f8452 github.com/klauspost/compress@v1.10.0`

Reply:
looks like they have widely used packages https://github.com/klauspost/compress

Reply:
I'm guessing some CPUs can't generate these keys properly https://github.com/klauspost/cpuid